centrallix-ui-test/tests/editbox_test.py

```diff
@@ -0,0 +1,132 @@
+""" module allowing web testing """
+
+import time
+import asyncio
+import toml
+import imagehash as ImageHash
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from PIL import Image
+from pyppeteer import launch
+
+def create_driver(test_url) -> webdriver:
+    """ Create a WebDriver """
+    service = Service(executable_path="chromedriver.exe")

```

Comment:
This should be a configuration option (chromedriver.exe executable path)

Similar for the other test.